src/app/GUI.py

```diff
@@ -85,28 +85,59 @@ def init(self):
     def _mapiraj(self, x: float, y: float) -> tuple[int]:
         return int(self.sirina * x), int(self.visina * y)
 
-    def _omejitev_pozicije(self):
+    # DODANO ZARADI ŠIRINE SLIKE
+    def _omejitev_pozicije_ladje(self):
         sirina = 1 - self.vesolje.ladja.velikost_x
         if self.vesolje.ladja.x >= sirina:
             self.vesolje.ladja.x = sirina
 
+    # DODANO ZARADI ŠIRINE SLIKE
+    def _omejitev_pozicije_vesoljcev(self):
+        # TODO: TUKAJ SEM ZAMENJAL self.vesolje.stev_vesoljcev[-1].velikost_x Z self.vesolje.velikost_vesoljca_x
+        #  ČE VELJA DA BOJO VSI VESOLJCI ISTE VELIKOSTI :/
+        sirina = 1 - self.vesolje.velikost_vesoljca_x
+        if self.vesolje.stev_vesoljcev[-1].x >= sirina:
+            for j in range(len(self.vesolje.stev_vesoljcev)):
+                self.vesolje.stev_vesoljcev[j].smer = S.APP.smer.levo
+                self.vesolje.stev_vesoljcev[j].y += self.vesolje.premik_vesoljca_navzdol
```

Comment:
`_omejitev_pozicije_ladje` in `_omejitev_pozicije_vesoljcev` bo treba spremeniti ker tukaj manipuliraš z domenskim prostorom... App layer mora samo prikazovati stanje domenskega prostora ne pa ga tudi spreminjat saj je potem zelo težko videti kaj kdo koga spreminja in kje se ta sprememba dogaja zato je pomembno da te dve funkciji implementiraš na način da ne boš spreminjal nobene stvari iz domenskega prostora.

Reply:
Edino kjer se sme manipulirati z domenskim prostorom je v funkciji vnos tam dejansko poveš kaj kakšne tipke na tipkovnici naredijo v domeni... Ampak to lahko samo tam narediš in nikjer drugje.

Reply:
Tu mi ene funkcije ni uspelo modificirat, tako da nebi spremenil domenskega prostora. @urosjarc